5-Server-ts/tododao.ts

```diff
@@ -0,0 +1,38 @@
+import redis, {RedisClient} from 'redis';
+import {TodoListDTO} from "../common/interfaces";
+
+const client: RedisClient = redis.createClient(process.env.REDIS_URL || "");
+
+
+export class TodoDao {
+    getTodos = (userId: string) => {
+        return new Promise<TodoListDTO>((resolve, reject) => {
```

Comment:
very nice turning it to promise. now I want u to write a function called `promisify` it can take any function that has the `callback` pattern and turn it into a promise based function. then use  `promisify` in your DAO